lib/gringotts/gateways/monei.ex

```diff
@@ -258,13 +359,11 @@ defmodule Gringotts.Gateways.Monei do
      "paymentBrand": card.brand]
   end
 
-  @doc """
-  Makes the request to MONEI's network.
-  """
+  # Makes the request to MONEI's network.
   @spec commit(atom, String.t, keyword, keyword) ::
```

Comment:
I think there is no need of  `@spec`, since it is private method.

Reply:
No harm in documenting though, it does not end up in the docs, and is for devs who happen to read this file.